test/mp4/impeghd_mp4_file_wrapper.c

```diff
@@ -152,6 +155,17 @@ WORD32 impeghd_mp4_fw_read(ia_file_wrapper *transport, pUWORD8 buffer, WORD32 bu
       if (err_test == 2)
       {
         *length = 0;
+        if (transport->is_mp4_file && transport->is_execution == 1)
```

Comment:
Is there any reason to add transport->is_execution == 1 condition?

Reply:
Instead of is_execution, is_initialized can we use?

Reply:
`is_execution` is alias for `is_initialized`.